### PR TITLE
Support SUSE distributions

### PR DESCRIPTION
**Pull request by guangyee**
Checking on whether a given package is installed is still being done with dpkg. For SUSE platforms, dpkg must be explicitly installed.

Signed-off-by: guangyee <gyee@suse.com>

### CODE REVIEW

File: scripts/script.sh [hunk 1]
```diff
@@ -61,8 +63,23 @@ set -o nounset
...
+        ;;
+    *)
+        echo "ERROR: OS distribution ${OS_DISTRIBUTION} is not supported."
+        exit 1
+esac
+
+
 ${PROG_WHICH:="which"} \
-    ${PROG_APT:="apt"} \
+    ${PKG_MGR} \
```
Comment (KeithMnemonic) [inline scripts/script.sh:82]: Is this correct here, with no assignment?  Is it needed?

Comment (guangyee) [reply scripts/script.sh:82]: Yes, it will still checking for the respective OS's package manager to make sure it exist.

**Comment by KeithMnemonic** [comment]
LGTM

**Review by KeithMnemonic — APPROVED** [review]
LGTM

**Pull request merged**